From 86337f3503c11ab599aa8ea3beb54f2155c21fa6 Mon Sep 17 00:00:00 2001
Message-Id: <86337f3503c11ab599aa8ea3beb54f2155c21fa6.1428954517.git.dvhart@linux.intel.com>
From: Darren Hart <dvhart@linux.intel.com>
Date: Mon, 13 Apr 2015 12:43:52 -0700
Subject: [PATCH] Remove dependencies from setup.py

Upstream-status: Inappropriate

Prevent the install from trying to download backports.ssl_match_hostname
and certifi and handle the dependency in the bitbake meta-data. This
avoids proxy issues with the setuptools installer.

Signed-off-by: Darren Hart <dvhart@linux.intel.com>
---
 setup.py | 14 ++++++++------
 1 file changed, 8 insertions(+), 6 deletions(-)

diff --git a/setup.py b/setup.py
index 963a6dd..e2f9293 100644
--- a/setup.py
+++ b/setup.py
@@ -118,12 +118,14 @@ if (platform.python_implementation() == 'CPython' and
         kwargs['cmdclass'] = {'build_ext': custom_build_ext}
 
 
-if setuptools is not None:
-    # If setuptools is not available, you're on your own for dependencies.
-    install_requires = ['certifi']
-    if sys.version_info < (3, 2):
-        install_requires.append('backports.ssl_match_hostname')
-    kwargs['install_requires'] = install_requires
+# FIXME: ABSOLUTE TOTAL HACK
+# Disable dependencies and handle them through the bitbake meta-data
+#if setuptools is not None:
+#    # If setuptools is not available, you're on your own for dependencies.
+#    install_requires = ['certifi']
+#    if sys.version_info < (3, 2):
+#        install_requires.append('backports.ssl_match_hostname')
+#    kwargs['install_requires'] = install_requires
 
 setup(
     name="tornado",
-- 
2.1.4

